gcode_metadata/__init__.py

```diff
@@ -18,5 +14,10 @@
 __all__ = [
     "MetaData", "FDMMetaData", "SLMetaData", "get_metadata",
     "UnknownGcodeFileType", "estimated_to_seconds", "get_preview", "get_icon",
-    "get_meta_class"
+    "get_meta_class",
 ]
+
+
+from .metadata import MetaData, FDMMetaData, SLMetaData, get_metadata, \
```

Comment:
Tohle ti poradil kdo? Takhle by to být nemělo, nejprve importuješ a pak definuješ